server/app/views/admin/programs/ProgramApplicationView.java

```diff
@@ -258,7 +258,8 @@ private DivTag renderStatusOptionsSelector(
         .map(StatusDefinitions.Status::statusText)
         .forEach(
             statusText -> {
-              boolean isCurrentStatus = statusText.equals(application.getLatestStatus().orElse(""));
+              String latestStatusText = application.getLatestStatus().orElse("");
```

Comment:
This can go above the entire stream operation so it's capture outside of the forEach context and not done each time.

Reply:
Done.